Sources/Workspace/Workspace.swift

```diff
@@ -110,80 +110,85 @@ public class Workspace {
     /// A struct representing all the current manifests (root + external) in a package graph.
     public struct DependencyManifests {
         /// The package graph root.
-        let root: PackageGraphRoot
+        internal let root: PackageGraphRoot
 
         /// The dependency manifests in the transitive closure of root manifest.
-        let dependencies: [(manifest: Manifest, dependency: ManagedDependency)]
+        internal let dependencies: [(manifest: Manifest, dependency: ManagedDependency)]
```

Comment:
I find that explicitly having it signifies it's a conscious choice to have that function available to the rest of the module and/or tests and not an omission of access control. For example, while reviewing all implicitly-internal functions in Workspace, I found that some should actually be private. I used the explicit internal modifier to "flag" the functions I had already asserted needed to be internal.